integration-tests.Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN yarn config set proxy "$http_proxy" && yarn config set https-proxy "$https_p
 
 COPY --chown=hmcts:hmcts package.json yarn.lock /usr/src/app/
 
-RUN PUPPETEER_SKIP_DOWNLOAD=true yarn install && yarn cache clean
+RUN yarn install && yarn cache clean
```

Comment:
was this change pushed as part of https://github.com/hmcts/cmc-citizen-frontend/pull/3394?

Reply:
it was part of #3395